app/derivative-services/external_metadata_derivative_service.rb

```diff
@@ -30,6 +30,12 @@ def cleanup_derivatives; end
   # Extract external geo metadata into parent vector or raster resource.
   def create_derivatives
     GeoMetadataExtractor.new(change_set: parent_change_set, file_node: change_set.resource, persister: persister).extract
+    # Send record updated message for parent resource
+    messenger.record_updated(parent)
```

Comment:
Maybe GeoMetadataExtractor should use a ChangeSet/ChangeSetPersister for the parent? Then it would fire that event.

Reply:
👍 That's definitely a better approach.